CMakeLists.txt

```diff
@@ -139,7 +144,7 @@ ENDIF()
 
 set(Kokkos_VERSION_MAJOR 4)
 set(Kokkos_VERSION_MINOR 0)
-set(Kokkos_VERSION_PATCH 0)
+set(Kokkos_VERSION_PATCH 01)
```

Comment:
```suggestion
set(Kokkos_VERSION_PATCH 1)
```

Reply:
@dalg24 fixed with [0ab1f11](https://github.com/kokkos/kokkos/pull/6080/commits/0ab1f11fe31ec7e180537252f4732a6c9573b27e)

Reply:
I should have mentioned the snapshot to Trilinos is also updated with the corrected patch version convention